packages/edit-site/src/components/sidebar/spacing-panel.js

```diff
@@ -123,6 +130,15 @@ export default function SpacingPanel( { context, getStyle, setStyle } ) {
 		setStyle( name, 'padding', padding );
 	};
 
+	const marginValues = getStyle( name, 'margin' );
+	const themeMarginValues = useThemeValues( name, 'margin' );
```

Comment:
I understand this has not been introduced in this PR, so happy to iterate on a different one.

I understand that the goal of `useThemeValues( name, property )` is to get the base values (core+theme). I believe is in our best interest to evolve getStyles to also allow retrieving these data such as: `getStyle( name, property, 'theme' )` instead of doing what we do at the moment. The rationale for that is that we want to centralize access to the shape of the data, so consumers don't need to deal with shape changes, migrations, etc. ― instead, there's a centralized place for all of that (global styles provider, getStyle method, etc).

Reply:
Yes, commented about it here https://github.com/WordPress/gutenberg/pull/30607#discussion_r617366595

Reply:
Oh, I saw your other comment now. So it looks like I should have started by reviewing that one.

Reply:
Thanks 👍 

`useThemeValues` has been removed and values are now simply unset. I'll create a new PR soon to update `getStyle` to support a `theme` origin.